### ISSUE TITLE
Question about training data structure

### ISSUE THREAD

**Issue by YuanSun-XJTU** [Wangt-CN/DisCo#12]
Hi, I would like to use a different dataset for the second step of fine-tuning. How should I structure the data as provided by you? For example, how can I obtain the files `train_images.lineidx` and `train_images.lineidx.8b`? 
Can you provide a brief tutorial on how to use `tsv_file_ops.py` and `tsv_file.py`?

**Comment by Wangt-CN** [comment]
Hi @YuanSun-XJTU , @kevinlin311tw will update our brief tutorial soon for the tsv file construction. Currently, you can check [this file](https://github.com/Wangt-CN/DisCo/blob/main/dataset/tiktok_controlnet_t2i_imagevar_combine_mask.py) as a work around by using the raw image/mask/pose from your new dataset for fine-tuning. Will let you know when the tsv tutorial is ready.

**Comment by YuanSun-XJTU** [comment]
THANKS!
